src/three/plugins/SUBTREELoader.js

```diff
@@ -0,0 +1,747 @@
+/**
+ * Structure almost identical to Cesium, also the comments and the names are kept
+ * https://github.com/CesiumGS/cesium
+ */
+import {Matrix3, Vector3} from 'three';
+import {SubtreeTile} from "../../base/SubtreeTile.js";
+import {LoaderBase} from '../../base/loaders/LoaderBase.js';
+import {readMagicBytes} from '../../utilities/readMagicBytes.js';
+import {arrayToString} from "../../utilities/arrayToString.js";
+
+
+export class SUBTREELoader extends LoaderBase {
+
+	constructor(tile, rootTile) {
+		super();
+		this.tile = tile;
+		this.rootTile = rootTile;	//The first tile build from the tileset
+
+	}
+
+	/**
+	 * A helper object for storing the two parts of the subtree binary
+	 *
+	 * @typedef {object} Subtree
+	 * @property {number} version
+	 * @property {JSON} subtreeJson
+	 * @property {ArrayBuffer} subtreeByte
+	 * @private
+	 */
```

Comment:
Is this a comment from a deleted function?

Reply:
No it's a typedef to define the output of `parseBuffer`. Isn't really useful but it can help for the understanding of the code in the IDE